docs/examples/promise/05_read_storage/index.html

```diff
@@ -361,6 +361,90 @@
             
         </li>
     
+        <li class="chapter " data-level="3.2" data-path="../../rx/">
```

Comment:
Not sure why this is in the PR, it is generated. Should be removed and CI should just do its job.